### DISCUSSION TITLE
Stabilisation de la rareté des familiers

### DISCUSSION THREAD

**Discussion by Evatco** [Crownicles/Crownicles#2360] [Nouvelles idées]
Je propose d'ajouter les pets 6 et 7 étoiles avec de nouvelles mécaniques et pour qu'ils restent éternellement rares. Je vous fais une version développée plus bas et en premier avec des tirets car c'est long :p
<br>_Familiers 6 étoiles :_
<br>**Acquisition :** Exclusivement disponibles dans les guildes de niveau supérieur à 100, avec une chance de 0.01% (ou 0.001%) par niveau de guilde. 
**Nourriture/Affection :** Nécessitent une affection atteignant le niveau 200, avec des paliers de progression correspondant à des pourcentages définis.
**Mécaniques spéciales :** Ces familiers peuvent exécuter de petites attaques pendant les combats, chacun possédant une attaque unique, permettant des combinaisons stratégiques en fonction des classes ou des situations.
**Avantages :** Ils pourraient avoir des avantages supplémentaires face aux boss ou une capacité spéciale à utiliser pendant les combats.
<br>_Familiers 7 étoiles :_
<br> **Acquisition :** Obtention via un nouveau processus nommé "Sacrifice". Il nécessite de sacrifier un familier de chaque étoile dressé, déclenchant un mini-événement pour compléter ce rituel.
**Propriété et Affection :** Ces familiers seraient liés au joueur et ne pourraient pas être vendus ou échangés. Ils pourraient être temporairement prêtés à d'autres membres de la guilde, mais leur affection diminuerait rapidement, les ramenant au propriétaire après deux mois.
**Mécaniques spéciales :** Similaires aux 6 étoiles avec de petites attaques en combat. En plus, ils auraient une très faible chance de bloquer les dégâts subis par le joueur lors de mini-événements ou d'événements, avec une limite de deux événements par jour.

<br><br>Version développée : 
<br>_______Pour que les 6 étoiles soit rares on pourrait les obtenir que dans les guilde au dessus de 100 où chaque niveau vaudrait 0.01% (ou 0.001% à voir) de chance. Il faudrait aussi les nourrir sur 200 et on garde les même proportions pour les différents stades
Après faut aussi ajouter un avantage comparé au 5 étoiles donc je propose qu'il y ait 5-6 pets 6 étoiles et qu'ils ont un faible chance de faire une petite attaque en combat, mais chaque pet fait une attaque différente et donc faire des combinaisons de pets en fonction des classes, ect. Après à voir s'ils ont un boost face aux boss ou s'il peuvent aussi utiliser la rage, ect
<br>_______Pour les 7 étoiles je propose une nouvelle mécanique le sacrifice !! Pour avoir un 7 étoiles faudrait sacrifier un pet de toutes les étoiles dressé (un 1 étoiles, un 2 étoiles, ..., un 6 étoiles) . Ce serait un mini évent quand on a pas les pets qu'il faut comme "vous voyez une statue représentant un grand esprit dans un cercle entouré de 6 pets différents, qu'est ce que ça pourrait être." Et quand c'est bon "la statue mystérieuse vous appelle, voulez vous y aller ? ". Ça pourrait donner un badge "a invoqué une créature mystique" mais par contre les 7 étoiles ne pourraient pas être vendu/échangé. Ils seraient lié au joueur mais ils pourrait être prêter à un membre de la guilde mais même dressé il perdrait beaucoup d'affection par jour et par exemple au bout de 2 mois il retournerait forcément vers le propriétaire. Bien sur il est possible de perdre le 7 étoile à cause de l'affection qui serait sur 400 ou 600. Je les voyaient plus comme des esprits invoqué plus que des pets. Ils auraient aussi les petites attaques des 6 étoiles mais ils auraient en plus une très faible probabilité et la possibilité de bloquer les dégâts fait au joueur dans les mini évents (ou évents ?), avec une limite de 2 mini évent par jour

**Comment by Pietagorh** [comment]
Sacrifice yay
Sinon ça me rappelle pas mal #573 donc on pourrait y piocher des idées niveau bonus

**Reply by Evatco** [reply]
C'est vrai qu'au niveau des bonus je me suis inspiré de posts d'avant mais les bonus c'est pas le plus important, c'est le plus factuel dans mon post

**Comment by BananePlantain** [comment]
Idée vraiment cool,c'est vrai que pour l'instant les pets à part quelques minis-events qui donne un peu de vie/argent/temps et pour la rage 1 fois sur 12 ainsi qu'une fois le badge 5* dressée obtenu leur utilité est limitée.Obtention des familiers 6* bien ça permet de donner une raison de plus aux joueurs de monter leur guilde niveau 100 tout en restant quand même très rare,**_il faut donc qu'il donne un avantage à la hauteur de la difficulté(encore plus pour les 7*)._** Obtention des 7* ça m'as l'air vraiment vraiment très compliqué donc faut que ça vaille le coup de payer une quantité astronomique de nourriture x6 (ou les dresser à coup de minis-events/bonbons faut la patience)+trouver un 3/4/5* sur la route +avoir une guilde niv100+ pour un 6* (à 1-1.5% chaud)(je compte que 11 guildes qui sont éligibles aux conditions actuellement (j'ai calculé à partir du classement guildes ,donc ayant fait l'ile y en a peut-être un peu plus)) donc éventuellement les acheter si tu n'as pas de chance (avec 1000 le 3*;5k le 4* et plus de 40k le 5* alors le 6* j’imagine pas) ça chiffre vite énormément.
PS:en plus pour le 6* faut le nourrir plus que les autres raretés pour atteindre le niveau "dressé" pauvre portefeuille 💸 à part pour Voltou complexe de se permettre ça x)

**Reply by Evatco** [reply]
Je suis d'accord que pour qu'une guilde ait complétement des 7 étoiles ça va couter cher mais au moins ils font rester rares mais les avantages liées au coup sont peut-être un peu bas :p
Ça peut être bien de baisser les coup mais les acheter fait perdre du charme à l'esprit invoqué donc soit avec beaucoup de gemmes ou que les 6 étoiles soient plus fréquents pour qu'il y en ait des ventes
je vois plutôt ça pour réguler la rareté 
Bon je referais le message car il est pas très compréhensible ^-^

**Reply by BananePlantain** [reply]
Ou à la place de 1 familier de chaque rareté pour le sacrifice juste 4/5/6* parce que dresser des famillers bas niveaux ca rapporte rien

**Reply by BananePlantain** [reply]
+perdre un familier 6/7 vu comment c'est galère rip (même si quand t' as farmer pour un tu en prends bien soin) imagines qu'il faille 9 d'affection pour le 6* et ptet 17 ou plus pour les 7* (pour qu'il soit plus fielleux et au moins sauvage) tu as le temps de tomber sur un mini-event "fuite de l'animal" (bien qu’extrêmement rare)quand il est fielleux

**Reply by Pietagorh** [reply]
> Ou à la place de 1 familier de chaque rareté pour le sacrifice juste 4/5/6* parce que dresser des famillers bas niveaux ca rapporte rien

Heh, j'aime bien le concept de mettre un de chaque, surtout que du coup tu dois faire de la place dans ton shelter pour les garder et ça peut être assez épique d'avoir dans son shelter un de chaque (ça ferait un shelter full du coup) pour obtenir un 7*
C'est un peu genre la culmination de tous tes efforts par rapport aux pets à toutes les étapes d'aventure qui se rejoignent c'est cool

**Reply by BananePlantain** [reply]
à la limite de chaque ok mais dressé 💸 et dans ce cas faut un refuge de guilde chacun 😶

**Reply by Pietagorh** [reply]
Mais justement c'est giga dur d'en obtenir donc t'façon c'est un effort global de tous les membres de la guilde donc j'aime bien l'idée de rendre le projet un minimum encombrant en termes d'espace de guilde

**Reply by Pietagorh** [reply]
Et puis bon les 1 2 3* ça coûte pas trop cher à nourrir à 6, surtout avec les gd d'une guilde level 100

**Reply by Evatco** [reply]
Pour le mini évent de fuite c'est sur que c'est problématique, faudrait qu'il ne soit pas possible avec un pet 7 étoiles pendant une semaine, ou 2 ou 1 mois ^^

**Reply by Pietagorh** [reply]
Voire même pas possible du tout
J'trouve juste ne pas avoir les avantages en plus suffirait déjà à être un gros désavantage quand t'as passé autant de temps et d'efforts à essayer d'avoir un pet aussi

**Reply by Evatco** [reply]
Oui je pensais aux gens qui aurraient un 7 étoiles et inactifs mais peu probables :p après faudrait aussi voir tout les problèmes que poserait que le pet soit la propriété du joueur, surtout si c'est toute la guilde qui dépense dedans. Mais c'est aussi interressant de laisser comme ça, cela obligerait que la guilde ait confiance en le joueur
Donc oui ça peut-être pas mal que ça soit impossible

**Reply by BananePlantain** [reply]
sauf que si le refuge est complet et que tu trouves un bon familier c'est la haagrah un peu

**Comment by Evatco** [comment]
> à la limite de chaque ok mais dressé 💸 et dans ce cas faut un refuge de guilde chacun 😶

Pour le refuge de guilde, comme tu peux prendre un pet sur toi, tu peux tout à fait garder un animal de ta guilde, après le problème est que si tu garde un pet dressé comment savoir s'il sera pris ou pas (deux pets même étoile dressé) mais dans ce cas là faudrait demander de choisir donc

**Reply by BananePlantain** [reply]
donc ?

**Reply by Evatco** [reply]
Je sais plus pourquoi j'ai mis un donc mais je crois que c'était pas forcément le plus pratique par contre ce serait peut-être mieux que tous les pets sacrifié soie,t dans l'abri de guilde et que le pet sur soi ne soit pas concerné par le sacrifice

**Reply by BananePlantain** [reply]
à la limite un /sacrifice qui affiche un tableau d'avancement des familiers sacrifiées et ceux manquants et /sacrifier qui permet de mettre le familier qu'on as sur soi dans l'autel de sacrifice (on peux transférer sans pb entre refuge et son familier de toute façon)

**Comment by BananePlantain** [comment]
> Et puis bon les 1 2 3* ça coûte pas trop cher à nourrir à 6, surtout avec les gd d'une guilde level 100

bah autant que les autres raretés si tu veux payer des soupes pour aller + vite

**Reply by Pietagorh** [reply]
> > Et puis bon les 1 2 3* ça coûte pas trop cher à nourrir à 6, surtout avec les gd d'une guilde level 100
> 
> bah autant que les autres raretés si tu veux payer des soupes pour aller + vite
> 

Déjà nan et en plus pourquoi t'utiliserais des soupes quand t'façon les 4/5* ça prend autant de temps de les dresser avec des soupes que les 1/2/3* avec des salades ou viandes

**Reply by BananePlantain** [reply]
c'est tjrs une question de temps si je veux dresser un 5* avec des bonbons rien ne m'en empêche mais c'est très long

**Comment by Akutana** [comment]
Je trouverais ça super qu'on modernise le système de pets, ça fait longtemps que ça n'a pas bougé. Les sacrifices c'est une bonne idée, par contre au niveau des guildes minimum lvl 100 ça fait vrmt bcp
D'un côté c'est bien pck ça demande de l'investissement et c'est pas histoire d'avoir de la chance, mais d'un autre côté ça serait donc réservé à "l'élite" donc seule une infime minorité pourrait en profiter
Après ça dépend comment on voit ces nouveaux pets, soit on décide que ça demande bcp de temps et d'investissement et demande donc d'être dans le top 5% des joueurs/guildes, soit on reste sur l'idée que chacun peut y arriver peu importe son niveau
Je balance une idée qui me vient en tête : on rend accessible les pets 6 et 7 à tt le monde, mais une fois qu'on a une guilde lvl 100 ou une condition comme ça on peut l'améliorer une fois qu'il est dressé en sacrifiant un autre pet, style 4 ou 5 pck 6 ça serait vrmt abusé

**Reply by Pietagorh** [reply]
> d'un autre côté ça serait donc réservé à "l'élite" donc seule une infime minorité pourrait en profiter

Ouais mais là t'façon les gens qui ont déjà des 5* dressés on parle de très très late game, et j'trouve que c'est nécessaire pour que ça reste rare
Sinon ça va juste énerver de voir des level 5 avec des 6* parce qu'ils ont juste eu de la chance alors que y'a que 20 personnes dans le bot qui en ont quoi

**Reply by Pietagorh** [reply]
Justement y'a beaucoup de gens qui se plaignent du manque de contenu en late game de ce que j'ai vu dans les dernières années donc donner un but giga dur et rare ça peut motiver
En plus y aura de plus en plus de guildes level 100 avec le temps donc ce sera plus acceptable genre l'an prochain que maintenant

**Reply by Akutana** [reply]
Ouais c'est pour ça que je suis partagé

**Reply by Akutana** [reply]
Hmm vu comme ça ouais c'est cohérent

**Comment by Procion06** [comment]
je préfère cette idée là #2138  moins usine à gaz et dont tout le monde peut profiter ^^

**Comment by BananePlantain** [comment]
on peut proposer un système d'entrainement du familier,permettant à "l'élite" d'avoir des avantages en plus contre de l'argent,j'aime bien sinon l'idée que je sois réservée au tryhardeurs qui ne savent pas où dépenser leur temps ou argent

**Reply by BananePlantain** [reply]
je re-re-reviens ici parceque y a plusieurs top joueurs qui n'ont plus d'objectif,qui tourne en rond etc et un peu de défi pour rebooster tout ca on prend
On pense moins à la rétention des gros joueurs n'empeche que y en a pleins qui font des pauses par ennui.

**Reply by BananePlantain** [reply]
et que maintenant que je relis la discussion je me dis que finalement c'est assez accessible quand je vois que 'jai de l'argent qui s'accumule vite

**Reply by BananePlantain** [reply]
et puis c'est quand même une part du jeu délaissé et qui pourrait apporter tellement plus
